lib/persisted-model.js

```diff
@@ -1489,7 +1489,6 @@ module.exports = function(registry) {
 
   function getIdFromWhereByModelId(Model, where) {
     var whereKeys = Object.keys(where);
-    if (whereKeys.length != 1) return undefined;
 
     var idName = Model.getIdName();
     if (whereKeys[0] !== idName) return undefined;
```

Comment:
I am out of the office until 01/04/2016.

Happy holidays!


Note: This is an automated response to your message  "Re: [loopback] WIP:
Rectify change vs rectify all changes [Optimization] (#1899)" sent on
12/21/15 9:25:50 AM.

This is the only notification you will receive while this person is away.